frb_codegen/src/library/commands/cargo_expand.rs

```diff
@@ -111,7 +116,12 @@ fn run_cargo_expand(
 
     let mut stdout_lines = stdout.lines();
     stdout_lines.next();
-    let ans = stdout_lines.join("\n").replace("/// frb_marker: ", "");
+    let ans = stdout_lines.join("\n");
+    static PATTERN: OnceLock<Regex> = OnceLock::new();
```

Comment:
nit: what about lazy_static (slightly shorter code)

Reply:
Personally I prefer this because no macros, but up to you

Reply:
Our codegen already uses many lazy_static, so I guess it is OK to be consistent